### PR TITLE
makes easier to find where to put the launch options for steam version v6

### PR DESCRIPTION
**Pull request by Joethepic**

### CODE REVIEW

File: worlds/v6/docs/setup_en.md [hunk 1]
```diff
@@ -13,10 +13,14 @@
 
 # Joining a MultiWorld Game
 
-To join, set the following launch options: `-v6ap_name YourName -v6ap_ip ServerIP:Port`.
-Optionally, add `-v6ap_passwd "YourPassword"` if the room you are using requires a password. All parameters without quotation marks.
-The Name in this case is the one specified in your generated .yaml file.
-In case you are using the Archipelago Website, the IP should be `archipelago.gg`.
+To join an Archipelago MultiWorld game, you must set the game's launch options. The two mandatory launch options are:
```
Comment (LegendaryLinux) [inline worlds/v6/docs/setup_en.md:16]: Add two spaces after this line. Doing to will cause the following code block to appear on the following line.

File: worlds/v6/docs/setup_en.md [hunk 2]
```diff
@@ -13,10 +13,14 @@
...
+`-v6ap_name slotName`
+`-v6ap_ip server:port`
```
Comment (LegendaryLinux) [inline worlds/v6/docs/setup_en.md:18]: Remove the blank like above this line and add two spaces to the end of the preceding code block. Doing so will cause markdown to insert a new line.

File: worlds/v6/docs/setup_en.md [hunk 3]
```diff
@@ -13,10 +13,14 @@
...
+
+If the game you are joining requires a password, you should also add the following to your launch options:
```
Comment (LegendaryLinux) [inline worlds/v6/docs/setup_en.md:20]: Add two spaces to the end of this line. Same reason as above.

File: worlds/v6/docs/setup_en.md [hunk 4]
```diff
@@ -13,10 +13,14 @@
...
+`-v6ap_passwd secretPassword`
+
+Launch options may be found by right-clicking on the game in Steam, and clicking "Properties" in the context menu. From there, open the "General" tab. There is a "Launch Options" setting near the bottom wherein you should enter the above options. Note that all launch options are seperated by a space, and if there are spaces in your password, it should be surrounded with quotes
```
Comment (LegendaryLinux) [inline worlds/v6/docs/setup_en.md:23]: 1. This paragraph's last sentence needs to end with a period.
2. Remove the comma after the word "Steam" in the first sentence.

**Review by LegendaryLinux — CHANGES_REQUESTED** [review]

File: worlds/v6/docs/setup_en.md (hunk 2 above)
Comment (N00byKing) [inline worlds/v6/docs/setup_en.md:18]: I'd prefer if this said `ServerIP:Port`

File: worlds/v6/docs/setup_en.md [hunk 5]
```diff
@@ -13,11 +13,14 @@
...
-The Name in this case is the one specified in your generated .yaml file.
-In case you are using the Archipelago Website, the IP should be `archipelago.gg`.
+To join an Archipelago MultiWorld game, you must set the game's launch options. The two mandatory launch options are:  
+  `-v6ap_name slotName`
+  `-v6ap_ip serverIP:port`
+If the game you are joining requires a password, you should also add the following to your launch options:
+  `-v6ap_passwd secretPassword`  
 
+
+Launch options may be found by right-clicking on the game in Steam and clicking "Properties" in the context menu. From there, open the "General" tab. There is a "Launch Options" setting near the bottom wherein you should enter the above options. Note that all launch options are seperated by a space, and if there are spaces in your password, it should be surrounded with quotes.
```
Comment (ThePhar) [inline worlds/v6/docs/setup_en.md:23]: Fixed a typo and also should clarify that slot names also need to be surrounded with quotes if there are spaces in them. :)

```suggestion
Launch options may be found by right-clicking on the game in Steam and clicking "Properties" in the context menu. From there, open the "General" tab. There is a "Launch Options" setting near the bottom wherein you should enter the above options. Note that all launch options are separated by a space, and if there are spaces in your slot name or password, it should be surrounded with quotes.
```

**Review by LegendaryLinux — APPROVED** [review]

**Pull request merged**